mongo-store.js

```diff
@@ -263,24 +263,17 @@ module.exports = function (opts) {
 
           coll.find(qq, mq, function (err, cur) {
             if (!error(args, err, cb)) {
-              var list = []
-
-              cur.each(function (err, entp) {
+              const resultsList = []
```

Comment:
refactor this in the same code style, naming, etc that it was originally rather than CLP style. Reasons being it keeps changes to a minimum and makes merging code from the original project easier.